src/test/java/test/Test.java

```diff
@@ -0,0 +1,93 @@
+package test;
```

Comment:
General convention,

When writing test for `ClassA`,
- The test name will typically be `ClassATest`.
- The package of the test class will be same as class `ClassA`.
- This can be done quickly by invoking generate(Alt+Insert ) > Test, if we have test frameworks available for the project.

Reply:
Fixed.